yt_dlp/extractor/pladform.py

```diff
@@ -63,13 +63,20 @@ def _real_extract(self, url):
             'http://out.pladform.ru/getVideo', video_id, query={
                 'pl': pl,
                 'videoid': video_id,
-            })
+            }, fatal=False)
 
         def fail(text):
             raise ExtractorError(
                 '%s returned error: %s' % (self.IE_NAME, text),
                 expected=True)
 
+        if not video:
+            targetUrl = self._request_webpage(url, None, note='Resolving final URL').geturl()
```

Comment:
```suggestion
            targetUrl = self._request_webpage(HEADRequest(url), video_id, note='Resolving final URL').geturl()
```

Reply:
HEAD request not allowed in tested cases

Reply:
You can revert that part then. But add the video_id

Reply:
Already done in latest commit